### PR TITLE
Bugfix/qa fixes

### PR DESCRIPTION
**Pull request by parunagiri**
Hi @backer,
Good morning! Have a nice day.

Please find below bugs has been fixed as part of this PR,

- [RBR-73](https://brightcove.atlassian.net/browse/RBR-73):  [Android] Previous video content is seen enlarged in background when next video is played and the player controls flicker
- [RBR-76](https://brightcove.atlassian.net/browse/RBR-76): [Android] Next video screen is not seen when user cancels next video screen and rewinds and fast forward to the last 5 seconds of the video
- [RBR-77](https://brightcove.atlassian.net/browse/RBR-77) : [Android] Next video screen shows the same video even when the user moves the scrubber back and to the last 5 seconds
- [RBR-78](https://brightcove.atlassian.net/browse/RBR-78) : [Android] Unable to play the previous video from the playlist after the next video plays

### CODE REVIEW

File: android/src/main/java/jp/manse/BrightcovePlayerView.java [hunk 1]
```diff
@@ -103,6 +108,17 @@ public void onPlayNext(Video video) {
             playVideo(video);
         }
     };
+    /**
+     * Please do use this variable any other purpose. it may lead into video size issue
```
Comment (backer) [inline android/src/main/java/jp/manse/BrightcovePlayerView.java:112]: I believe this comment should say "Please do not use"

File: android/src/main/java/jp/manse/BrightcovePlayerView.java [hunk 2]
```diff
@@ -103,6 +108,17 @@ public void onPlayNext(Video video) {
...
+     * one of condition to refresh the video size
+     **/
+    private int prevOrientationForRefreshVideoLayout = Configuration.ORIENTATION_PORTRAIT;
+
+    /**
+     * Please do use this variable any other purpose. it may lead into video size issue
```
Comment (backer) [inline android/src/main/java/jp/manse/BrightcovePlayerView.java:118]: same here

**Review by backer — APPROVED** [review]
just fix the two comments and then this is good to go 👍

File: android/src/main/java/jp/manse/BrightcovePlayerView.java (hunk 1 above)
Comment (parunagiri) [reply android/src/main/java/jp/manse/BrightcovePlayerView.java:112]: Yes, @backer. Apologies for the spelling mistake

**Pull request merged**